Fix typo on configuration template file

sebasfalcone opened the pull request:
|Related issue|
|---|
|#16657|

## Description
This PR aims to fix a badly formatted configuration file involved in the installation process.

All the analysis is on [the issue](https://github.com/wazuh/wazuh/issues/16657#issuecomment-1504130828).

## CL
- Fixed typo on the `etc/ossec-local.conf`
- Added warning message to indicate that the configurations provided on the script were not successfully applied (using default configurations on `etc/ossec-local.conf`)

## Tests
- [ ] Source installation
  - [x] Linux
  - [ ] MAC OS X

kevin-mcm commented on src/init/inst-functions.sh:
```diff
@@ -904,6 +904,7 @@ InstallCommon()
         if [ -f  ../etc/ossec.mc ]; then
             ${INSTALL} -m 0660 -o root -g ${WAZUH_GROUP} ../etc/ossec.mc ${INSTALLDIR}/etc/ossec.conf
         else
+            echo "ERROR: unable to generate ossec.conf file with desired configurations, using default configurations from ${OSSEC_CONF_SRC}"
```
Question: Isn't it better to use a **WARNIG** message?
```suggestion
            echo "WARNING: unable to generate ossec.conf file with desired configurations, using default configurations from ${OSSEC_CONF_SRC}"
```

sebasfalcone replied:
I think you are right! I will change it

sebasfalcone commented:
# Tests

## Preparations
In order to test the changes we need to perform two changes:
- Fix the local compilation (this is due to a bug on the vm_database.c file)
  - The fix is to replace that file following one [vm_database.c.zip](https://github.com/wazuh/wazuh/files/11950885/vm_database.c.zip)
- Add a stop to the installation script (in order to remove the ossec.mc file):
https://github.com/wazuh/wazuh/blob/71026b63d12b5abbd0d7d7904b1ec9072ab1b288/src/init/inst-functions.sh#L932-L934

Those lines should look like this:
```
    fi
    
    $ECHO "Press any key to continue...  "
    read ANSWER

    if [ ! -f ${INSTALLDIR}/etc/ossec.conf ]; then
```

## What to expect
The ossec.mc file is a temporary file generated based on the configurations provided in the installation script. The configurations I've provided are:
```
  3.1- Do you want e-mail notification? (y/n) [n]: y
   - What's your e-mail address? sebas@sebas.com
   
  3.5- Active response allows you to execute a specific
       command based on the events received.
       By default, no active responses are defined.

   - Default white list for the active response:
      - 127.0.0.53

   - Do you want to add more IPs to the white list? (y/n)? [n]: y
   - IPs (space separated): 1.1.1.1
```

The rest of the configurations are the default ones.

The script checks if the `ossec.mc` file was generated and if it was it copies its content to `/var/ossec/etc/ossec.conf`, if the file is not present, then it uses the default configuration (the file I've fix)

In order to test the script I've added the user input stop and removed the `ossec.mc` file.

### Ossec.mc file (important sections)
```
  ....

  <global>
    <jsonout_output>yes</jsonout_output>
    <alerts_log>yes</alerts_log>
    <logall>no</logall>
    <logall_json>no</logall_json>
    <email_notification>yes</email_notification>
    <smtp_server>in2-smtp.messagingengine.com.</smtp_server>
    <email_from>wazuh@development</email_from>
    <email_to>sebas@sebas.com</email_to>
    <email_maxperhour>12</email_maxperhour>
    <email_log_source>alerts.log</email_log_source>
    <agents_disconnection_time>10m</agents_disconnection_time>
    <agents_disconnection_alert_time>0</agents_disconnection_alert_time>
  </global>

  ...

  <global>
    <white_list>127.0.0.1</white_list>
    <white_list>^localhost.localdomain$</white_list>
    <white_list>127.0.0.53</white_list>
    <white_list>1.1.1.1</white_list>
  </global>

  ...
```
Both IP `1.1.1.1` and `sebas@sebas.com` fields are present

### ossec.conf vs ossec-local.conf
```
root@development: ~
🍄 diff /var/ossec/etc/ossec.conf /home/vagrant/wazuh/etc/ossec-local.conf 
root@development: ~
```

Both files are the same and the script warned us of this (its a bit masked by other logs):
```
Press any key to continue...  
WARNING: unable to generate ossec.conf file with desired configurations, using default configurations from ../etc/ossec-local.conf
```
